.github/workflows/integration_tests.yml

```diff
@@ -351,10 +351,10 @@ jobs:
         if: matrix.target_platform == 'Desktop' && !cancelled()
         run: |
           python scripts/gha/desktop_tester.py --testapp_dir ta
-      - name: Run iOS integration tests on Simulator locally
-        if: contains(env.mobileTestOn, 'simulator') && matrix.target_platform == 'iOS' && !cancelled()
+      - name: Run iOS integration tests on Simulator/Emulator locally
```

Comment:
Yeah.. it's a little bit confusing. But if we only use "simulator", it would be confusing that the test script has a lot "emulator" command.
I will come up with a better name in another PR.